### PR TITLE
Adobe flash cheat sheet

### PR DESCRIPTION
**Pull request by ankushg07**
https://duck.co/ia/view/adobe_flash_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@ankushg07 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Adobe flash cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/adobe_flash_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Adobe flash cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/adobe_flash_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/adobe-flash.json [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+			"key": "[Enter]",
+			"val": "Play"
+		}, {
+			"key": "[Ctrl] [0]",
+			"val": "Rewind"
+		}, {
+			"key": "[<]",
+			"val": "Previous frame"
+		}, {
+			"key": "[>]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/adobe-flash.json]: @ankushg07 for single keys, they don’t need to be wrapped in `[]`. The brackets are used to separate individual keys when the string contains more than one.

Please remove the `[]` around the single keys. You shouldn't see and `[]` when the Cheat Sheet renders: https://beta.duckduckgo.com/?q=Adobe+flash+cheat+sheet&ia=answer&iax=1

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Adobe flash cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/adobe_flash_cheat_sheet_mobile.png)

**Comment by ankushg07** [comment]
@moollaza formatted the code please have a look

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Adobe flash cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/adobe_flash_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/adobe-flash.json [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+			"key": "E",
+			"val": "Eraser"
+		}, {
+			"key": "H",
+			"val": "Hand"
+		}, {
+			"key": "Z",
+			"val": "Magnifier"
+		},{
+            "key": "Q",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/adobe-flash.json]: @ankushg07 looks like the indentation is still incorrect -- you're using hard-tabs still

File: share/goodie/cheat_sheets/json/adobe-flash.json [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+			"val": "Delete Frame"
+		}, {
+			"key": "F6",
+			"val": "Add keyframe"
+		}, {
+			"key": "F7",
+			"val": "Add blank keyframe"
+		}],
+		"Timeline": [{
+			"key": "[Enter]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/adobe-flash.json]: This one still has brackets!

Make sure you check visually ;)

**Comment by ankushg07** [comment]
@moollaza indentation was fine on codio thats why it went unnoticed, Formatted via online validator, looks fine now

**Comment by ankushg07** [comment]
@moollaza @GuiltyDolphin any update??

**Comment by moollaza** [comment]
@ankushg07 Thanks and apologies for the delay! This LGTM :+1:

**Pull request merged**